.travis/opensuse.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash -xe
-${DOCKER_EXEC} zypper -n ar -f -G http://download.opensuse.org/repositories/devel:/tools/openSUSE_13.2/devel:tools.repo
+OS_VERSION=$(${DOCKER_EXEC} cat /etc/os-release | awk -F[=\"] '/PRETTY_NAME=/ {print $3}')
```

Comment:
While this change will work for openSUSE leap 42.2 the same change may not work if 13.2 were to come back again.
On 13.2 we get - PRETTY_NAME="openSUSE 13.2 (Harlequin) (x86_64)"
There isn't anything I could find on how they create their urls but there is a pattern of combining "NAME" and "VERSION_ID" which results into their url. 
You might want to combine NAME and VERSION_ID instead of relying on PRETTY_NAME.
What do you think?